lib/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+# TODO since this is a library, we need to depend on actual published versions.
```

Comment:
Same as before, I think we can remove this TODO.

Also for `fil_builtin_actors_bundle` we will have to rely on Git as latest published version on crates.io is 7.5.1. And I do not think that they plan to release it soon based on [their README](https://github.com/filecoin-project/builtin-actors/#releasing)